lib/routes/situations.js

```diff
@@ -39,4 +39,7 @@ module.exports = function(api) {
 
     api.route('/situations/:situationId/openfisca-request').get(allowShow, situations.openfiscaRequest);
 
+    api.route('/situations/:situationId/openfisca-test').get(situations.openfiscaTest);
+    api.route('/situations/:situationId/openfisca-test').post(situations.openfiscaTest);
```

Comment:
You can chain declarations on a [`route`](http://expressjs.com/fr/api.html#router.route), no need to duplicate the route pattern.